admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/integration/MasterdataIntegrationTest.java

```diff
@@ -2004,6 +2004,7 @@ public void addHolidayTypeLanguageValidationTest() throws Exception {
 				.andExpect(status().isOk());
 	}
 
+	@Ignore

```

Comment:
why you are skipping the test case

Reply:
there are some issues with the Json we are passing, updated the same, and removed it from the ignored test cases.